lib/streamlit/config.py

```diff
@@ -454,6 +455,17 @@ def _logger_message_format() -> str:
     scriptable=True,
 )
 
+_create_option(
+    "client.snowparkMaxRows",
```

Comment:
This probably needs to be named slightly more generic so that it can be eventually used by other unevaluated dataframe implementations (e.g. Spark Dataframe) (fyi @jrieke). Also, in general, we try to don't include too many snowflake-specific aspects in the Streamlit core codebase.

Reply:
As of a new spec this is outdated 👍